packages/remix-react/routeModules.ts

```diff
@@ -71,7 +71,16 @@ export interface MetaFunction {
  * `name` attribute.
  */
 export interface HtmlMetaDescriptor {
-  [name: string]: string | string[] | null | undefined;
+  charset?: "utf-8";
+  charSet?: "utf-8";
```

Comment:
@MichaelDeBoey A little late to the party here, but unless I'm wrong, we shouldn't force `utf-8` here.
`windows-1252` is valid for example.

Reply:
👍🏼  (I was looking at an older version of the HTML spec.)